tests/common/dualtor/mux_simulator_control.py

```diff
@@ -167,7 +167,7 @@ def _post(server_url, data):
         server_url = '{}?reqId={}'.format(server_url, uuid.uuid4())  # Add query string param reqId for debugging
         logger.debug('POST {} with {}'.format(server_url, data))     # lgtm [py/clear-text-logging-sensitive-data]
         headers = {'Accept': 'application/json', 'Content-Type': 'application/json'}
-        resp = requests.post(server_url, json=data, headers=headers)
+        resp = requests.post(server_url, json=data, headers=headers, timeout=600)
```

Comment:
@StormLiangMS http request only takes 1 or 2 seconds to get response. After discussed with Xin, he suggested to use 5s, if 5s, there is no response, it will have no response in 10m as well. I change it to 10s to have much buffer in case of any network delay.